### ISSUE TITLE
Deal with mis-configured forms

### ISSUE THREAD

**Issue by moshthepitt** [onaio/kaznet-web#310]
Make it possible to:

- [ ] identify forms that are not within the expected Onadata organisation
- [ ] identify forms that do not have the right level (can submit) permissions for members of the Kaznet organisation
- [ ] move a form to the right organization
- [ ] change permissions of a project so that all members of the Kaznet organization can make submissions

**Comment by ukanga** [comment]
Where are this endpoints? Kaznet only or onadata API endpoints?

I will probably say no. Is the XForm not in project's `forms` list?

**Comment by moshthepitt** [comment]
This is data from Onadata API.

The form is in the Project's XForm list, but we want to check:

- the form's owner is the expected organization
- the members of that organisation have the "dataentry" role for the project that the form belongs to

**Comment by lincmba** [comment]
```
teams: [
{
    role: "owner",
    name: "ptorg#Owners",
    users: [
        "linc"
]
},
{
    role: "editor",
    name: "ptorg#members",
    users: [
        "ptorg",
        "testjane",
        "linc",
        "winny",
        "simba"
]
}
]```

Hey @moshthepitt . Here is the teams value at the projects endpoint. Given `ptorg` organisation `linc` as owner and the other users as editors

**Comment by moshthepitt** [comment]
@lincmba so is it reliable to conclude that in my example above members of the kaznet organisation can definitely submit data to that form just based on:

- the form owner ( "owner": "kaznet")
- that the form owner exists in the teams of the project
- that the role of that team is at "dataentry"

**Comment by lincmba** [comment]
Yes. Members with `dataentry` role will submit data to the forms in that project.

**Comment by moshthepitt** [comment]
```python
form_owner = 'kaznet' 
form_members_team = project['teams'].get(f"{form_owner}#members")
form_correctly_configured = form_members_team is not None and form_members_team['role'] == "dataentry"
```

@lincmba @ukanga  is this reliable?

**Comment by lincmba** [comment]
`organization` in place of `form_owner` is more accurate.
